install_scripts/templates/common/docker-version.sh

```diff
@@ -146,7 +146,11 @@ getMaxDockerVersion() {
     fi
     # Amazon has their own repo and 1.12.6 and 17.06.2 are available there for now.
```

Comment:
Do we want to change this comment? Maybe something along the lines of `Amazon Linux prior to 17.03 has Docker 17.06.02 available, as does Amazon Linux 17.06. Amazon Linux 17.03 only has Docker 17.03.02.`